lesson2/homework/helloworld/src/main/java/MoneySaver.java

```diff
@@ -27,12 +27,12 @@ public String toString() {
 
     public void deposit(long deposit_amount) {
         myAccount.deposit_money(deposit_amount);
+        this.myMoney -= deposit_amount;
     }
 
     public void withdraw(long withdraw_amount) {
-         myAccount.withdraw_moolah(withdraw_amount);
-        //((CheckingAccount(myAccount)).withdraw_moolah(withdraw_amount);
-
+        myAccount.withdraw_moolah(withdraw_amount);
+        this.myMoney += withdraw_amount;
```

Comment:
Similarly, add the money back into your pocket after you withdraw it from your account